routers/api/v1/user/app.go

```diff
@@ -199,6 +184,109 @@ func DeleteAccessToken(ctx *context.APIContext) {
 	ctx.Status(http.StatusNoContent)
 }
 
+// ListAccessTokens list all the access tokens
+func ListAccessTokensDeprecated(ctx *context.APIContext) {
+	// swagger:operation GET /users/{username}/tokens user userGetTokensDeprecated
```

Comment:
As an admin I would like it more to use this instead of an `admin/users` endpoint. I know I'm an admin, I don't need every of my actions behind `admin/`.

Reply:
Right, but for compatibility reasons I don't want to change the existing API behavior at the moment, so it's labeled `Deprecated`. Then the admin action is placed after `admin/`.

Reply:
Just label it as bugfix "`/users/{username}/tokens` does not respect `username` parameter" and breaking.